builder/vmware/common/driver_workstation_unix.go

```diff
@@ -116,26 +116,23 @@ func workstationNetmapConfPath() string {
 	return filepath.Join(base, "netmap.conf")
 }
 
-func workstationToolsIsoPath(flavor string) string {
-	return "/usr/lib/vmware/isoimages/" + flavor + ".iso"
+func workstationToolsIsoPath(appPath, flavor string) string {
+	libPath := filepath.Dir(filepath.Dir(appPath))
```

Comment:
Could you add a small comment to state that `appPath` is expected to point to the `vmware` tool installed on the machine?
That'd make it easier to understand the process of inferring the lib path.

The implementation looks OK to me otherwise

Reply:
Ok, added here: f108aed

Reply:
Perfect, thanks